src/server.h

```diff
@@ -67,6 +67,7 @@ typedef long long ustime_t; /* microsecond time type. */
 #include "ae.h"         /* Event driven programming library */
 #include "sds.h"        /* Dynamic safe strings */
 #include "dict.h"       /* Hash tables */
+#include "hashset.h"    /* Hash set */
```

Comment:
Is the hashset.c/.h file in a separate commit/PR ? I don't see the file in unstable. Or am I missing something ?

Reply:
This is targeting the hashset branch which has a lot of work done by Viktor. :)

Reply:
Ah, my bad.